web/app/view/map/BaseMap.js

```diff
@@ -187,6 +187,27 @@ Ext.define('Traccar.view.map.BaseMap', {
 
         this.map.addControl(new ol.control.LayerSwitcher());
 
+        popup = new ol.Overlay.Popup();
+        this.map.addOverlay(popup);
+
+        geocoder = new Geocoder('nominatim', {
+            provider: 'bing',
+            key: server.get('bingKey'),
+            lang: 'en-US',
+            placeholder: 'Search for ...',
+            targetType: 'glass-button',
+            limit: 5,
+            autoComplete: true,
+            keepOpen: true
+        });
+        this.map.addControl(geocoder);
+
+        geocoder.on('addresschosen', function (evt) {
+            window.setTimeout(function () {
+                popup.show(evt.coordinate, evt.address.formatted);
+            }, 3000);
```

Comment:
Why is there a timeout?

Reply:
The timeout was taken from the sample code -- now removed